### DISCUSSION TITLE
Rendere la grafica che assomigli a classeviva

### DISCUSSION THREAD

**Discussion by CobraOfficial01** [iisgiua/giuaschool#763] [Idee e proposte]
Fare un registro tipo Classeviva se si può

**Comment by robertopadovesi** [comment]
Sinceramente, anche no. L'interfaccia non è una delle migliori, preferisco molto di più Giua! inoltre, fammi notare questa cosa: Classeviva è pieno di funzioni e talvolta anche in modo disornato e vasto, sì, anche Giua c'è l'ha, ma ha l'occorrente necessario per una scuola e un istituto.

**Comment by trinko** [comment]
Ricordo che l'aspetto grafico può essere personalizzato sui template tramite CSS.

**Comment by CobraOfficial01** [comment]
Come faccio a trovare i file css?

**Reply by trinko** [reply]
> Come faccio a trovare i file css?

Li trovi nella cartella **public/css** ma è meglio modificare i singoli template che ti interessano usando gli stili in linea ([vedi esempio](https://supporthost.com/it/imparare-css/#css-in-linea-css-inline))

**Comment by robertopadovesi** [comment]
Li trova nel file manager del Registro Elettronico e nei database SQL, solitamente è lì.

**Reply by federicabianchi08** [reply]
Esatto, ClasseViva è piuttosto obsoleto come registro. Sicuramente è funzionale e offre molte più funzioni rispetto ad altri, ma è organizzato davvero male e ha un layout che sembra molto datato.

**Comment by federicabianchi08** [comment]
Esatto, ClasseViva è piuttosto obsoleto come registro. Sicuramente è funzionale e offre molte più funzioni rispetto ad altri, ma è organizzato davvero male e ha un layout che sembra molto datato.
